### PR TITLE
Added support for RFC proxies + reading VOMS extensions

### PR DESCRIPTION
**Pull request by acasajus**
NEW: Added support for RFC proxies throughout DIRAC
NEW: Read voms information in proxies directly without using voms-proxy-info
BACKPORT: Added LICENSE print as required by GPLv3
BACKPORT: Reloader. If the --autoreload flag is enabled, if there's a change in the python code the running process will automatically reload. 

WARNING: Requires pyGSI 0.6.2 (set in externals commit 0509155)

### CODE REVIEW

File: Core/Utilities/Devloader.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
+
```
Comment (sposs) [inline Core/Utilities/Devloader.py]: I know you aren't into documentation, but could you add a little something that says what it is and what it does?

**Comment by fstagni** [comment]
Did you mean for v6r13, right?

**Comment by atsareg** [comment]
No, next one

File: Core/Utilities/Devloader.py (hunk 1 above)
Comment (acasajus) [inline Core/Utilities/Devloader.py]: I've updated the comment box for the PR. If you're developing a service/whatever that's long running and you run it with --autoreload. If the python code changes the running process will autoreload.

**Comment by fstagni** [comment]
So, anyway this is targeted to integration branch. If it is for v6r14, should be anyway targeted to the v6r13 branch

**Comment by acasajus** [comment]
I'll close this and issue a new PR then.

**Comment by acasajus** [comment]
Created #2272 targeting r13 branch

**Pull request closed without merging**